images/plbase/python-requirements.txt

```diff
@@ -26,3 +26,4 @@ PuLP==2.4
 text-unidecode==1.3
 statsmodels==0.12.2
 openpyxl==3.0.7
+automata-lib==5.0.0
```

Comment:
We are trying to avoid adding more specialized libraries to the base image. Instead, you can install this yourself into the course `serveFilesCourse` directory and import it directly from question code. Would that work for you as an alternative?
```suggestion
```

Reply:
Could you elaborate on how to install it into `serverFilesCourse`? I'm not super familiar with how pip works.